lib/radiator/feed/episode_builder.ex

```diff
@@ -60,6 +62,18 @@ defmodule Radiator.Feed.EpisodeBuilder do
     nil
   end
 
+  defp contributors(%Episode{audio: %Audio{contributors: contributors}}) do
+    contributors
+    |> Enum.filter(fn %Person{display_name: name} ->
+      String.valid?(name) && String.length(name) > 0
+    end)
+    |> Enum.map(fn contributor ->
+      element(:"atom:contributor", [element(:"atom:name", contributor.display_name)])
+    end)
```

Comment:
Since the `atom:contributor` is a `atom:atomPersonConstruct` it also has an uri, email and possible extensions. I think the Person should have at least one uri to actually supply something here. According to `https://tools.ietf.org/html/rfc4287#section-6.4` simple extensions there are possible, so I would encourage the introduction of a `<atom:role>` child element as well.

Reply:
Should we have a distinction between public and private emails then? Because the system needs you to supply an email to work but that doesn't necessarily mean that the user wants this email to be public.

Reply:
Yes we do want a distinction. Another reason to keep the account data in the Auth.User and the public facing description of a person in the Person